packages/paste-core/components/list/src/index.tsx

```diff
@@ -1,5 +1,5 @@
-export * from "./List";
-export * from "./ListItem";
-export * from "./OrderedList";
-export * from "./UnorderedList";
-export * from "./types";
+export { List } from "./List";
+export { ListItem } from "./ListItem";
+export { OrderedList } from "./OrderedList";
+export { UnorderedList } from "./UnorderedList";
+export type { BaseListProps, ListItemProps, OrderedListProps, UnorderedListProps } from "./types";
```

Comment:
suggestion: `BaseListProps` is kind of redundant information when ordered and unordered lists are documented below. Do we use it else where outside of the package? If not we can probably not export it so only real components have API docs

Reply:
All good to remove, exporting that was a mistake